apps/desktop-wallet/public/electron.js

```diff
@@ -312,6 +315,20 @@ app.on('ready', async function () {
     }
   })
 
+  ipcMain.handle('app:restart', () => {
+    app.relaunch()
+    app.exit()
+  })
+
+  ipcMain.handle('app:clearCache', async () => {
+    const session = mainWindow.webContents.session
+
+    await session.clearCache()
+    await session.clearStorageData({
```

Comment:
Should we keep this if this doesn't solve our issue? I'm a bit scared of potential unforeseen side effects. WDYT?

Reply:
ok, we can remove it.

Reply:
pushed